### ISSUE TITLE
only some songs are transferred to my playlist  

### ISSUE THREAD

**Issue by marcxxb** [Arturo254/InnerTune#20]
when I transfer songs from my YouTube music playlist (which contains 349 songs) only 99 songs go on my playlist.  Furthermore, some songs disappear from the playlist for no reason.  I even managed to transfer the 349 songs, but with little time the number of songs decreases, not significantly, but it decreases. 
I'm using version 0.6.1 on Android 14.
https://github.com/Arturo254/InnerTune/assets/71355507/27aa2036-c4dd-4c51-a939-04b7ccbd98a3

**Comment by Arturo254** [comment]
Don't try to create a new playlist, just press the icon before the 3 dots to export the complete playlist

**Comment by marcxxb** [comment]
> Don't try to create a new playlist, just press the icon before the 3 dots to export the complete playlist

Thanks, I got it. But what about the songs disappearing? The playlist I created and showed in the previous video had 99 songs and before I sent all the songs to another existing playlist it had 97 songs. 
By the way, congratulations on your project, I'm loving the app.  
![Screenshot_20240523_004455_InnerTune.jpg](https://github.com/Arturo254/InnerTune/assets/71355507/62a3d103-6816-4941-a64e-c8149131915a)
